JMS/README.md

```diff
@@ -109,6 +109,29 @@ Which will download and copy the dependencies into the current directory.
 
 
 ### Building the samples with maven
```

Comment:
This isn't specific to maven, so maybe better to have the title as "Building and running the samples on windows", containing the separator guidance and a maven specific sub-section referring to the symbolic link.